src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +147,1102 @@ metadata_pkg.(
+reflects the proximity of that observation to training observations
+associated with that class, and how far away the observation is from those
+associated with other classes. Specifically, the distances, in the transformed
+(projected) space, of a new observation, from the centroid of each target class,
+is computed; the resulting vector of distances (times minus one) is passed to a
+softmax function to obtain a class probability prediction. Here "distance"
+is computed using a user-specified distance function.
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitypes with `schema(X)`
+- `y`: is the target, which can be any `AbstractVector` whose element
+  scitype is `OrderedFactor` or `Multiclass`; check the scitype
+  with `scitype(y)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Hyper-parameters
+
+- `normalize=true`: Option to normalize the between class variance for the number of
+   observations in each class, one of `true` or `false`.
+- `out_dim`: The dimension of the transformed space to be used by `predict` and
+   `transform` methods, automatically set if 0 is given (default).
+- `dist=Distances.SqEuclidean()`: The distance metric to use when performing
+   classification (to compare the distance between a new point and centroids in
+   the transformed space); must be a subtype of `Distances.SemiMetric` from
+   Distances.jl, e.g., `Distances.CosineDist`.
+
+
+# Operations
+
+- `transform(mach, Xnew)`: Return a lower dimensional projection of the input `Xnew` having the same scitype as `X` above.
+- `predict(mach, Xnew)`: Return predictions of the target given
+  features `Xnew` having the same scitype as `X` above. Predictions
+  are probabilistic but uncalibrated.
+- `predict_mode(mach, Xnew)`: Return the modes of the probabilistic predictions
+   returned above.
+
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `projection_matrix`: The matrix used to project `X` into a lower dimensional space.
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `explained_variance_ratio`: The ratio of explained variance to total variance. Each dimension corresponds to an eigenvalue.
+- `classes`: The classes seen during model fitting.
+- `class_means`: The matrix comprised of class-specific means as
+  columns (see above).
+- `mean`: The mean of the untransformed training data, of length `in_dim`.
+- `class_weights`: The weights of each class.
+- `nc`: The number of classes directly observed in the training data (which can be
+   less than the total number of classes in the class pool)
+
+# Examples
+
+```
+using MLJ
+
+SLDA = @load SubspaceLDA pkg=MultivariateStats
```

Comment:
For consistency across examples, let's just use the same name every time:
```suggestion
SubspaceLDA = @load SubspaceLDA pkg=MultivariateStats
```